gradle.properties

```diff
@@ -202,7 +202,8 @@ jsonSimpleVersion=1.1.1
 
 jtdsVersion=1.3.1
 
-jtidyVersion=r918
+# https://mvnrepository.com/artifact/org.jsoup/jsoup
```

Comment:
This comment shouldn't be necessary.  Generally, the comments in this file provide explanation for why a particular version is chosen if it deviates from the latest version for some reason.